File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/account_risk_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class AccountRiskVO:
    """账号风险检测响应"""
    created_at: str = None
    updated_at: str = None
    user_id: str = None
    device_id: str = None
    risk_level: str = None
    risk_score: int = None
    risks: List[RiskItem] = None
    suggestion: str = None
    need_verification: bool = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/AccountRiskVO.kt
package com.sdkwork.app

data class AccountRiskVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val userId: String? = null,
    val deviceId: String? = null,
    val riskLevel: String? = null,
    val riskScore: Int? = null,
    val risks: List<RiskItem>? = null,
    val suggestion: String? = null,
    val needVerification: Boolean? = null
)
